packages/shared/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@callstack/react-native-legal-shared",
+  "version": "0.1.1",
+  "description": "Shared code for all packages",
+  "main": "build/index",
+  "module": "build/index",
+  "types": "build/index.d.ts",
+  "exports": {
+    ".": {
+      "source": "./src/index.ts",
+      "types": "./build/index.d.ts",
+      "default": "./build/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "src",
+    "build",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__",
+    "!**/.*"
+  ],
+  "scripts": {
+    "test": "jest",
+    "typecheck": "tsc",
+    "lint": "eslint \"**/*.{js,ts,tsx}\"",
+    "clean": "del-cli build",
+    "build": "tsc -p tsconfig.json"
+  },
+  "keywords": [
+    "nodejs",
+    "oss",
+    "license",
+    "acknowledgements",
+    "license-management"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/callstackincubator/react-native-legal.git",
+    "directory": "packages/license-kit"
+  },
+  "author": "Mateusz Mędrek <luigimario1913@gmail.com> (https://github.com/mateusz1913)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/callstackincubator/react-native-legal.git/issues"
+  },
+  "homepage": "https://callstackincubator.github.io/react-native-legal/",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "dependencies": {
+    "glob": "^7.1.3"
+  },
+  "devDependencies": {
+    "@types/glob": "^8.1.0",
+    "@types/jest": "^29.5.5",
+    "del-cli": "^5.1.0",
```

Comment:
it is used only in `clean` command, let's replace it with `rimraf` to decrease amount of dependencies

Reply:
Done in 9391aa3066c594d2aa6abd181b2533b3484ea973